### DISCUSSION TITLE
pi-Base Community Call Thu 2024 March 21 2pm Eastern

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#557] [Announcements]
**note time change from 3pm Eastern to 2pm Eastern** see discussoin below

Agenda:

- STDC followup (and intros from new folks?)
- Homepage examples: https://github.com/orgs/pi-base/discussions/549
- Web enhancement: https://github.com/pi-base/web/pull/133
- AI and the pi-Base?? ([NSF funding opportunity](https://new.nsf.gov/funding/opportunities/artificial-intelligence-formal-methods))

Zoom link: https://southalabama.zoom.us/j/93833246214

**Comment by ccaruvana** [comment]
I unfortunately have some work conflicts with this one. I did however take a quick look at the STDC notes and I enjoy the preview of the diagram generation. Exciting!

**Reply by StevenClontz** [reply]
Oh just saw this - would March 28 work better?

**Reply by ccaruvana** [reply]
Replying in the newer thread.

**Comment by StevenClontz** [comment]
Hi folks, I double-booked this time and cannot make it. 🤦 

Would 2pm or 4:30pm Eastern instead work? Or I could do the usual 3pm Eastern time on March 28.

**Reply by ccaruvana** [reply]
I could do 2pm Eastern on the 21st or the usual 3pm on the 28th.

**Reply by prabau** [reply]
Both 2pm on the 21st and 3pm on the 28th work for me.

**Reply by StevenClontz** [reply]
Okay - 2pm Eastern 21st works for our highest-attendance community members, so let's do that.

**Comment by StevenClontz** [comment]
Note: will change homepage to use https://topology.pi-base.org/theorems/T000040/

**Comment by prabau** [comment]
@StevenClontz In case you have not seen this, https://us11.campaign-archive.com/?e=35e41bc9c4&u=9d7ced8c4bbd6c2f238673f0f&id=1f342c6283
Some example of use of generative AI when doing software development.  It may give some ideas for your proposal.

The main thing I get from this is that the human is still completely in charge.  The AI (GitHub Copilot) gives suggestions that speed up the development, by accepting the change in the editor (vscode) and then modify it manually as needed, which ends up being faster than having to type a bunch of boring/standard things from scratch.  So, in parallel, I am starting to understand the value of a pi-base "paper scraper" that would get a first pass at extracting a preliminary list of concepts and related results from a document, and then a human would pick and choose from that and modify as needed to speed up the overall process.  (Still not completely clear if it would be worth it, but I could see the benefit in theory.)
